Makefile

```diff
@@ -5,8 +5,9 @@ KUSTOMIZE_VERSION ?= 3.7.0
 GATEKEEPER_VERSION ?= release-3.5
 BATS_VERSION ?= 1.3.0
 GATOR_VERSION ?= 3.7.0
+GOMPLATE_VERSION ?= 3.10.0
 
-integration-bootstrap:
+integration-bootstrap: yq
```

Comment:
```suggestion
integration-bootstrap: yq gomplate
```

Reply:
gomplate is not a prerequisite of integration test so I don't need we need to add it here.